e2e/map.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+import intro from '../client/i18n/locales/english/intro.json';
+
+import {
+  SuperBlockStages,
+  superBlockOrder
+} from '../shared/config/superblocks';
+test.beforeEach(async ({ page }) => {
+  await page.goto('/learn');
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
```

Comment:
```suggestion
```

We actually don't need this. Playwright closes the page automatically.

Reply:
done